contracts/optimistic-ethereum/OVM/verification/OVM_StateTransitioner.sol

```diff
@@ -33,7 +33,7 @@ import { Abs_FraudContributor } from "./Abs_FraudContributor.sol";
  * This contract controls the State Manager and Execution Manager, and uses them to calculate the
  * post-state root by applying the transaction. The Fraud Verifier can then check for fraud by comparing
  * the calculated post-state root with the proposed post-state root.
- * 
+ *
```

Comment:
Lots of little white space fixes to ignore due to the addition of an `.editorConfig`.